app/javascript/src/QuickStarts/templates/createApplication.yaml

```diff
@@ -3,7 +3,7 @@ metadata:
   name: creating-an-application-quick-start
   # you can add additional metadata here
   instructional: true
-  learningResources: true
+  basicAPIIntegrationSetup: true
```

Comment:
I'm not a huge fan of "hardcoding" these categories.
I imagine having something like `category: basic-api-integration-setup`
And using this metadata in `CustomCatalog`